common/define-grammar.js

```diff
@@ -728,7 +728,7 @@ module.exports = function defineGrammar(dialect) {
         )),
         field('arguments', $.arguments)
       ),
-      type_query: $ => prec.right(seq(
+      type_query: $ => prec.right(1, seq(
```

Comment:
The tree-sitter-typescript grammars use the newer system for defining precedences, see the [`precedences` section](https://github.com/tree-sitter/tree-sitter-typescript/blob/1b3ba31c7538825b05815f4f5bffcca6394edc63/common/define-grammar.js#L19-L60). I forgot how this works but hopefully you can figure this out.